src/isgbd/model/FlowerVersionDao.java

```diff
@@ -0,0 +1,48 @@
+package isgbd.model;
+
+import com.mysql.jdbc.jdbc2.optional.MysqlDataSource;
+
+import java.sql.*;
+
+/**
+ * Created by Laura on 1/24/2018
+ */
+
+public class FlowerVersionDao {
+    private Connection conn;
+    private FlowerDao flowerDao = new FlowerDao();
```

Comment:
the flower version dao access only flower versions, the flower dao should be not used here